src/data/STATUSES/root/GNB.ts

```diff
@@ -50,6 +50,24 @@ export const GNB = ensureStatuses({
 		icon: iconUrl(13618),
 		duration: 10000,
 	},
+	READY_TO_RAZE: {
+		id: 3839,
+		name: 'Ready to Raze',
+		icon: iconUrl(13620),
+		duration: 10000,
+	},
+	READY_TO_BREAK: {
+		id: 3886,
+		name: 'Ready to Break',
+		icon: iconUrl(13622),
+		duration: 20000,
```

Comment:
This should be 30s (the No Mercy buff is only 20, but Ready to Break is 30 per the job guide). Unless the job guide is incorrect, in which case ignore me.